CHANGELOG.md

```diff
@@ -1,25 +1,26 @@
 
-## 1.0.0 (Unreleased)
-
-(https://github.com/vmware/terraform-provider-vmc/tree/HEAD)
-
-[Full Changelog](https://github.com/vmware/terraform-provider-vmc/compare/87cfcf8f21600ef6198389c569d1e988ca30b5a9...HEAD)
+## 1.1.0 (Unreleased)
 
 FEATURES:
 
-* resource/vmc_sddc: Support for CRUD operations on Software Defined Data Center on VMC.
-* data_source/vmc_org: Support to retrieve details about the organization.
-* data_source/vmc_connected_accounts: Support to retrieve a list of connected accounts in the given organization.
-* data_source/vmc_customer_subnets: Support to retrieve a customer's compatible subnets for account linking.
+* **New Resource:** `vmc_sddc`
+* **New Resource:** `vmc_public_ip` [\#43](https://github.com/vmware/terraform-provider-vmc/pull/43)
+* **New Data Source:** `vmc_org`
+* **New Data Source:** `vmc_connected_accounts`
+* **New Data Source:** `vmc_customer_subnets`
 
 
 ENHANCEMENTS:
 
-* resource/vmc_sddc: Added nsxt_reverse_proxy_url to SDDC resource data. [\#23](https://github.com/vmware/terraform-provider-vmc/pull/23)
-* data_source/vmc_connected_accounts: Added filtering to return AWS account ID associated with the account number provided in configuration. [\#30](https://github.com/vmware/terraform-provider-vmc/pull/30)
-* resource/vmc_public_ip: Added public IP resource. [\#43](https://github.com/vmware/terraform-provider-vmc/pull/43)
+* vmc_sddc: Added nsxt_reverse_proxy_url to SDDC resource data. [\#23](https://github.com/vmware/terraform-provider-vmc/pull/23)
+* vmc_connected_accounts: Added filtering to return AWS account ID associated with the account number provided in configuration. [\#30](https://github.com/vmware/terraform-provider-vmc/pull/30)
+* provider.go: Added org_id as a required parameter in terraform schema. [\#38](https://github.com/vmware/terraform-provider-vmc/pull/38)
+* data_source_vmc_customer_subnets.go : Adding validateFunctions for sddc and customer subnet resources. [\#41](https://github.com/vmware/terraform-provider-vmc/pull/41)
```

Comment:
Adding to added

Reply:
Should we mention changes made for terraform v0.12 compatibility?

Reply:
The v0.12 compatibility should be implied, but if you think i might effect uses, it should be included.

Reply:
Sounds good, we can leave it out.